testing/ssh-tools/APKBUILD

```diff
@@ -0,0 +1,24 @@
+# Contributor: Sven Wick <sven.wick@gmx.de>
+# Maintainer: Sven Wick <sven.wick@gmx.de>
+
+pkgname=ssh-tools
+pkgver=1.5
+pkgrel=0
+pkgdesc="collection of various tools using ssh"
+arch="noarch"
+url="https://github.com/vaporup/ssh-tools/"
+license="GPL-3.0"
+#depends="bash ncurses openssh-client colordiff"
+depends="bash cmd:ssh cmd:ssh-keygen cmd:ssh-keyscan cmd:tput"
+source="$pkgname-$pkgver.tar.gz::https://github.com/vaporup/${pkgname}/archive/v${pkgver}.tar.gz"
+options="!check"
+builddir="$srcdir/$pkgname-$pkgver"
+
+package() {
+	cd "$builddir"
+
+    mkdir -p $pkgdir/usr/bin/
+    install ssh-* $pkgdir/usr/bin/
```

Comment:
```suggestion
    install -D ssh-* "$pkgdir/usr/bin/"
```

Reply:
If I remove the builddir and mkdir lines you suggested abov, I get an error

> ssh-tools: Entering fakeroot...
> install: can't stat 'ssh-*': No such file or directory